docs/docs/cmd/spo/folder/folder-roleassignment-add.mdx

```diff
@@ -20,13 +20,19 @@ m365 spo folder roleassignment add [options]
 : The server- or site-relative decoded URL of the folder.
 
 `--principalId [principalId]`
-: The SharePoint principal id. It may be either an user id or group id for which the role assignment will be addd. Specify either `upn`, `groupName` or `principalId` but not multiple.
+: The SharePoint principal id. It may be either an user id or group id for which the role assignment will be addd. Specify either `upn`, `groupName`, `principalId`, `entraGroupId` or `entraGroupName` but not multiple.
 
 `--upn [upn]`
-: The upn/email of the user. Specify either `upn`, `groupName` or `principalId` but not multiple.
+: The upn/email of the user. Specify either `upn`, `groupName`, `principalId`, `entraGroupId` or `entraGroupName` but not multiple.
 
 `--groupName [groupName]`
-: The Microsoft Entra or SharePoint group name. Specify either `upn`, `groupName` or `principalId` but not multiple.
+: The Microsoft Entra or SharePoint group name. Specify either `upn`, `groupName`, `principalId`, `entraGroupId` or `entraGroupName` but not multiple.
```

Comment:
We should correct that now it relates to SharePoint Group only so we should dropt the first part and maybe leave it like 
```suggestion
: The SharePoint group name to add. Specify either `upn`, `groupName`, `principalId`, `entraGroupId` or `entraGroupName` but not multiple.
```